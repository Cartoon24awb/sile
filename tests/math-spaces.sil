\begin[class=plain]{document}

\script[src=packages/math]

Positive space commands:

\begin{math}
  \mi{thin}\thinspace
  \mi{med}\medspace
  \mi{thick}\thickspace
  \mi{en}\enspace
  \mi{quad}\quad
  \mi{qquad}\qquad
  \mi{end}
\end{math}

\par
\medskip
Positive space commands (aliases):

\begin{math}
  \mi{thin}\,
  \mi{med}\>
  \mi{thick}\;
  \mi{en}\enskip
  \mi{end}
\end{math}

\par
\medskip
Positive spaces in subscripts:

\begin{math}
  x_{
    \mi{thin}\thinspace
    \mi{med}\medspace
    \mi{thick}\thickspace
    \mi{en}\enspace
    \mi{quad}\quad
    \mi{qquad}\qquad
    \mi{end}_{
      \mi{thin}\thinspace
      \mi{med}\medspace
      \mi{thick}\thickspace
      \mi{en}\enspace
      \mi{quad}\quad
      \mi{qquad}\qquad
      \mi{end}
    }
  }
\end{math}

\medskip
Negative space commands:

\begin{math}
  \mi{nthin}\negthinspace
  \mi{nmed}\negmedspace
  \mi{nthick}\negthickspace
  \mi{end}
\end{math}

\medskip
Negative space commands (alias):

\begin{math}
  \mi{nthin}\!
  \mi{end}
\end{math}

\medskip
Negative spaces in subscripts:

\begin{math}
  x_{
    \mi{nthin}\negthinspace
    \mi{nmed}\negmedspace
    \mi{nthick}\negthickspace
    \mi{end}_{
      \mi{nthin}\negthinspace
      \mi{nmed}\negmedspace
      \mi{nthick}\negthickspace
      \mi{end}
    }
  }
\end{math}

\end{document}
