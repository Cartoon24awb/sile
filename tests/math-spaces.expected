Set paper size 	595.275597	841.8897729
Begin page
Mx 	49.7638
My 	49.6628
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 82 86 76 87 76 89 72 w=32.5342	(Positive)
Mx 	84.9855
T	86 83 68 70 72 w=22.7393	(space)
Mx 	110.4123
T	70 82 80 80 68 81 71 86 w=44.6338	(commands)
Mx 	155.0460
T	29 w=2.2900	(:)
Mx 	49.7638
My 	61.6628
Set font 	Libertinus Math;10;400;;normal;;LTR
T	85 73 74 79 w=16.6700	(nil)
Mx 	68.1005
Mx 	7.9000
Mx 	4.5400
Mx 	5.0600
T	78 a=7.9000 70 a=4.4700 69 a=5.0600	(nil)
Mx 	87.7915
T	85 73 74 68 76 w=20.6500	(nil)
Mx 	111.3164
T	70 79 w=9.8900	(nil)
Mx 	126.2064
T	82 86 66 69 w=19.9700	(nil)
Mx 	156.1764
T	82 82 86 66 69 w=25.0000	(nil)
Mx 	201.1764
T	70 79 69 w=14.9500	(nil)
Mx 	49.7638
My 	79.6628
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 82 86 76 87 76 89 72 w=32.5342	(Positive)
Mx 	84.9815
T	86 83 68 70 72 w=22.7393	(space)
Mx 	110.4043
T	70 82 80 80 68 81 71 86 w=44.6338	(commands)
Mx 	157.7217
T	11 w=3.1689	(()
Mx 	160.8906
T	68 79 76 68 86 72 86 w=26.9434	(aliases)
Mx 	187.8340
T	12 w=3.1689	())
Mx 	191.0029
T	29 w=2.2900	(:)
Mx 	49.7638
My 	91.6628
Set font 	Libertinus Math;10;400;;normal;;LTR
T	85 73 74 79 w=16.6700	(nil)
Mx 	68.1005
Mx 	7.9000
Mx 	4.5400
Mx 	5.0600
T	78 a=7.9000 70 a=4.4700 69 a=5.0600	(nil)
Mx 	87.7998
T	85 73 74 68 76 w=20.6500	(nil)
Mx 	111.3455
T	70 79 w=9.8900	(nil)
Mx 	126.2355
T	70 79 69 w=14.9500	(nil)
Mx 	49.7638
My 	109.6628
Set font 	Gentium Plus;10;400;;normal;;LTR
T	51 82 86 76 87 76 89 72 w=32.5342	(Positive)
Mx 	84.9842
T	86 83 68 70 72 86 w=26.6016	(spaces)
Mx 	114.2719
T	76 81 w=8.2275	(in)
Mx 	125.1857
T	86 88 69 86 70 85 76 83 87 86 w=41.7432	(subscripts)
Mx 	166.9289
T	29 w=2.2900	(:)
Mx 	49.7638
My 	121.6628
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2735 w=5.4100	(nil)
Mx 	54.3638
My 	123.7628
Set font 	Libertinus Math;8;400;;normal;;LTR
T	85 73 74 79 w=13.3360	(nil)
Mx 	69.0331
Mx 	6.3200
Mx 	3.6320
Mx 	4.0480
T	78 a=6.3200 70 a=3.5760 69 a=4.0480	(nil)
Mx 	84.7796
T	85 73 74 68 76 w=16.5200	(nil)
Mx 	103.5835
T	70 79 w=7.9120	(nil)
Mx 	115.4955
T	82 86 66 69 w=15.9760	(nil)
Mx 	139.4715
T	82 82 86 66 69 w=20.0000	(nil)
Mx 	175.4715
T	70 79 69 w=11.9600	(nil)
Mx 	187.4315
My 	125.4428
Set font 	Libertinus Math;6;400;;normal;;LTR
T	85 73 74 79 w=10.0020	(nil)
Mx 	198.4335
Mx 	4.7400
Mx 	2.7240
Mx 	3.0360
T	78 a=4.7400 70 a=2.6820 69 a=3.0360	(nil)
Mx 	210.2433
T	85 73 74 68 76 w=12.3900	(nil)
Mx 	224.3462
T	70 79 w=5.9340	(nil)
Mx 	233.2802
T	82 86 66 69 w=11.9820	(nil)
Mx 	251.2622
T	82 82 86 66 69 w=15.0000	(nil)
Mx 	278.2622
T	70 79 69 w=8.9700	(nil)
Mx 	49.7638
My 	141.4152
Set font 	Gentium Plus;10;400;;normal;;LTR
T	49 72 74 68 87 76 89 72 w=36.2793	(Negative)
Mx 	88.7302
T	86 83 68 70 72 w=22.7393	(space)
Mx 	114.1566
T	70 82 80 80 68 81 71 86 w=44.6338	(commands)
Mx 	158.7903
T	29 w=2.2900	(:)
Mx 	49.7638
My 	153.4152
Set font 	Libertinus Math;10;400;;normal;;LTR
T	79 85 73 74 79 w=22.0900	(nil)
Mx 	70.1871
Mx 	5.4200
Mx 	7.9000
Mx 	4.5400
Mx 	5.0600
T	79 a=5.4200 78 a=7.9000 70 a=4.4700 69 a=5.0600	(nil)
Mx 	90.8634
T	79 85 73 74 68 76 w=26.0700	(nil)
Mx 	114.2767
T	70 79 69 w=14.9500	(nil)
Mx 	49.7638
My 	171.4152
Set font 	Gentium Plus;10;400;;normal;;LTR
T	49 72 74 68 87 76 89 72 w=36.2793	(Negative)
Mx 	88.7272
T	86 83 68 70 72 w=22.7393	(space)
Mx 	114.1505
T	70 82 80 80 68 81 71 86 w=44.6338	(commands)
Mx 	161.4684
T	11 w=3.1689	(()
Mx 	164.6373
T	68 79 76 68 86 w=18.4619	(alias)
Mx 	183.0992
T	12 w=3.1689	())
Mx 	186.2682
T	29 w=2.2900	(:)
Mx 	49.7638
My 	183.4152
Set font 	Libertinus Math;10;400;;normal;;LTR
T	79 85 73 74 79 w=22.0900	(nil)
Mx 	70.1871
T	70 79 69 w=14.9500	(nil)
Mx 	49.7638
My 	201.4152
Set font 	Gentium Plus;10;400;;normal;;LTR
T	49 72 74 68 87 76 89 72 w=36.2793	(Negative)
Mx 	88.7289
T	86 83 68 70 72 86 w=26.6016	(spaces)
Mx 	118.0162
T	76 81 w=8.2275	(in)
Mx 	128.9296
T	86 88 69 86 70 85 76 83 87 86 w=41.7432	(subscripts)
Mx 	170.6727
T	29 w=2.2900	(:)
Mx 	49.7638
My 	213.4152
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2735 w=5.4100	(nil)
Mx 	54.3638
My 	215.5152
Set font 	Libertinus Math;8;400;;normal;;LTR
T	79 85 73 74 79 w=17.6720	(nil)
Mx 	70.7024
Mx 	4.3360
Mx 	6.3200
Mx 	3.6320
Mx 	4.0480
T	79 a=4.3360 78 a=6.3200 70 a=3.5760 69 a=4.0480	(nil)
Mx 	87.2401
T	79 85 73 74 68 76 w=20.8560	(nil)
Mx 	105.9626
T	70 79 69 w=11.9600	(nil)
Mx 	117.9226
My 	217.1952
Set font 	Libertinus Math;6;400;;normal;;LTR
T	79 85 73 74 79 w=13.2540	(nil)
Mx 	130.1766
Mx 	3.2520
Mx 	4.7400
Mx 	2.7240
Mx 	3.0360
T	79 a=3.2520 78 a=4.7400 70 a=2.6820 69 a=3.0360	(nil)
Mx 	142.5799
T	79 85 73 74 68 76 w=15.6420	(nil)
Mx 	156.6217
T	70 79 69 w=8.9700	(nil)
Mx 	295.2916
My 	780.6177
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20 w=4.6924	(1)
End page
Finish
